specs/pvs/corla.pvs

```diff
@@ -1031,7 +1113,8 @@ END dashboard
 % @trace Principly RDPR-6-Jul-2017 Section 25.2.2
```

Comment:
line 1023/1105 change to "Department of State Dashboard", as "State-wide Dashboard" might be mistaken for the public dash (which is also state-wide).